content/applications/sales/sales/gelato.rst

```diff
@@ -4,4 +4,166 @@
+  :guilabel:`order_status_updated` ensures Odoo receives order changes automatically.
+- :guilabel:`Method`: Click into the field and select the :guilabel:`HTTP Post` option, as this is
+   the request method used to send data from Gelato to Odoo.
+- Tick the checkbox next to :guilabel:`I want to take Authorization to this webhook`.
+- :guilabel:`Header Name`: In this field, type in `signature` to match the field in Odoo.
+- Click :guilabel:`Generate Key` to generate a :guilabel:`Header Value`.
+- Click :guilabel:`Create` to complete this webhook configuration.
+
+.. image:: gelato/gelato-webhook.png
+   :alt: Newly configured webhook in the Gelato platform.
+
+.. tip::
+   Copy and paste the API key and webhook on a notepad before tabbing out of the Gelato webpage as
+   backup.
+
+Configure Gelato connector in Odoo
+----------------------------------
+
+In Odoo, navigate to :menuselection:`Sales app --> Configuration --> Settings`, then scroll to the
+:guilabel:`Connectors` section. Enable the :guilabel:`Gelato` connector by ticking the checkbox.
+Next, paste the newly generated API keys and Webhook secret key into their respective fields. Once
+saved, Gelato is available in Odoo **Sales** and **eCommerce** products.
+
+Synchronizing Gelato products with Odoo Sales
+=============================================
+
+It is recommended to have products already configured in Gelato before configuring them in Odoo. To
+get the product ID in Gelato, navigate to the :guilabel:`Templates` page from the side bar menu.
+Select which product to synchronize in Odoo, then hover over the product card to reveal the
+:icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)` menu icon. Click the menu icon, then click
+:guilabel:`Copy Template ID` to copy the product template ID to the clipboard.
+
+.. seealso::
+   `Start selling products with Gelato: Quick & easy setup
+   <https://www.gelato.com/blog/get-started-with-gelato-creating-products>`_
+
+
+Odoo Sales Product
+------------------
+
+To create a product in Odoo that matches the Gelato one, navigate to :menuselection:`Sales app -->
+Products --> Products`, select :guilabel:`New` to create a new product form. Type in the product
+:guilabel:`Name`, then navigate to the :guilabel:`Sales` tab. Find the :guilabel:`Gelato` section,
+then click into the :guilabel:`Template Reference` field and paste the copied template ID from the
+Gelato product. Finally, click :guilabel:`Synchronize`.
+
+Successful synchronization pulls the Gelato product variant options into the newly configured Odoo
+product.
+
+In the new :guilabel:`Print Images` field, click the :guilabel:`default` marker to set a default
+product image. Click the :icon:`fa-pencil` :guilabel:`(edit)` icon and select the product image file
+to upload, then :guilabel:`Save & Close`.
+
+.. important::
+   The `Print Images` field *must* be configured on the product before it can be ordered.
```

Comment:
I believe this should use guilabel formatting? 
```suggestion
   The :guilabel:`Print Images` field *must* be configured on the product before it can be ordered.
```